overlord/auth/auth.go

```diff
@@ -64,6 +64,16 @@ type UserState struct {
 	StoreDischarges []string `json:"store-discharges,omitempty"`
 }
 
+// idOnly returns a *UserState with only the identification information
+// from u.
+func (u *UserState) idOnly() *UserState {
+	return &UserState{
+		ID:       u.ID,
+		Username: u.Username,
+		Email:    u.Email,
+	}
```

Comment:
It seems `idOnly` is not explicitly tested.

Reply:
added this now